snooty.toml

```diff
@@ -39,7 +39,6 @@ php-library = "MongoDB PHP Library"
 
 [constants]
 php-library = "MongoDB PHP Library"
-driver-short = "PHP library"
```

Comment:
S: I might have suggested removing this in the past, but I think it is actually good to have in here, even as a repeat. There are some sharedincludes that we might use that might use this source constant. Maybe you can just do this:
`driver-short = "{+library-short+}"`